test/uiTesting/mainPage.test.js

```diff
@@ -0,0 +1,57 @@
+import { expect } from "chai";
+const puppeteer = require("puppeteer");
+
+const baseUrl = "https://tbd.mattmerr.com";
```

Comment:
We'll want to make this localhost or something, right now it's opening browser to tbd.mattmerr.com. I'm not sure how we can launch a jekyll server while in the test though. Maybe exec "bundle exec jekyll serve" with an unusual port?

Reply:
http://127.0.0.1:4000 is the default localhost address of jekyll
  to run this test, have to open another terminal to lanch the distrition to localhost
 the command is `jekyll serve`